diff -urN fish-4.0.2/tests/checks/string.fish fish-4.0.2-fix-tests/tests/checks/string.fish
--- fish-4.0.2/tests/checks/string.fish	2025-04-20 14:11:52.000000000 +0100
+++ fish-4.0.2-fix-tests/tests/checks/string.fish	2025-05-06 13:49:25.124464688 +0100
@@ -1045,12 +1045,12 @@
 string shorten -m6 (set_color blue)s(set_color red)t(set_color --bold brwhite)rin(set_color red)g(set_color yellow)-shorten | string escape
 # Renders like "strin…" in colors
 # Note that red sequence that we still pass on because it's width 0.
-# CHECK: \e\[34ms\e\[31mt\e\[1m\e\[37mrin\e\[31m…
+# CHECK: \e\[34ms\e\[31mt\e\[1m\e\[97mrin\e\[31m…
 
 # See that colors aren't counted in ellipsis
 string shorten -c (set_color blue)s(set_color red)t(set_color --bold brwhite)rin(set_color red)g -m 8 abcdefghijklmno | string escape
 # Renders like "abstring" in colors
-# CHECK: ab\e\[34ms\e\[31mt\e\[1m\e\[37mrin\e\[31mg
+# CHECK: ab\e\[34ms\e\[31mt\e\[1m\e\[97mrin\e\[31mg
 
 set -l str (set_color blue)s(set_color red)t(set_color --bold brwhite)rin(set_color red)g(set_color yellow)-shorten
 for i in (seq 1 (string length -V -- $str))
diff -urN fish-4.0.2/tests/test_functions/isolated-tmux-start.fish fish-4.0.2-fix-tests/tests/test_functions/isolated-tmux-start.fish
--- fish-4.0.2/tests/test_functions/isolated-tmux-start.fish	2025-04-20 14:11:52.000000000 +0100
+++ fish-4.0.2-fix-tests/tests/test_functions/isolated-tmux-start.fish	2025-05-06 13:51:56.966201537 +0100
@@ -2,13 +2,7 @@
     set -l tmpdir (mktemp -d)
     cd $tmpdir
 
-    begin
-        echo 'set -g mode-keys emacs'
-        # macOS lacks the tmux-256color terminfo, use screen-256color instead.
-        if test (uname) = Darwin
-            echo 'set -g default-terminal "screen-256color"'
-        end
-    end >./.tmux.conf
+    echo 'set -g mode-keys emacs' >.tmux.conf
 
     function isolated-tmux --inherit-variable tmpdir
         # tmux can't handle session sockets in paths that are too long, and macOS has a very long
